src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -409,7 +410,16 @@ private async Task ProcessActionsAsync(LazyInput lazyInput, IContext context, Ac
                         {
                             actionTrace.ParsedSettings = settings;
                         }
-                        
+
+                        if (action is RedirectAction)
```

Comment:
The FlowManager should not know specifics for its actions. This logic should not be here.